guardian/typerighter
# Upgrade typerighter cdk dependencies

## What does this change?

This upgrades typerighter's cdk dependencies, and should resolve some [vulnerabilities](https://metrics.gutools.co.uk/d/fdib3p8l85jwgd/dependency-vulnerabilities-by-team?orgId=1&var-repo_owner=digital-cms&var-repo_owner=digital-cms-admins&from=1715677005954&to=1715698605954) from older libraries.

## How to test

CDK tests should pass and synth successfully. 

Deploy to CODE. The generated Cloudformation template should successfully update the rule database to use a 'PRIVATE_WITH_EGRESS' subnet. 

The database should work the same as before - test by searching some rules on https://manager.typerighter.code.dev-gutools.co.uk.

We should also be able to [connect to the database locally](https://github.com/guardian/typerighter#connecting-to-the-rule-manager-database-in-code-or-prod).

## Tests complete

- [x] CDK synths successfully
- [x] CDK tests pass
- [x] Deploy to CODE
- [x] Test rule search on CODE
- [x] Connect locally to CODE DB
